src/app/config.service.ts

```diff
@@ -45,6 +49,9 @@ export class ConfigService {
     enabled: false,
     time: 1000,
   };
+  @LocalStorage() _appearanceConfig: AppearanceConfig = {
+    borderThickness: 1
```

Comment:
We normally set these to the defaults, but here you've set it to a different value, is there a reason?

Reply:
Nope, this should have been a two